tests/app/index.html

```diff
@@ -115,5 +115,8 @@
       const template = dm.querySelector('template');
       assert(template.hasAttribute('css-build'), 'dom-module template should have css-build attribute');
     });
+
+    suite('class elements', function() {
```

Comment:
Left-over testing stuff?

Reply:
Sorry, made the PR before finishing the tests :/